src/components/Reports/MinisterHousingAllowance/NewRequest/NewRequestPage.tsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from 'react';
+import { Container, Stack } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useNewStepList } from '../../../../hooks/useNewStepList';
+import { mainContentWidth } from '../MinisterHousingAllowance';
+import { PanelLayout } from '../PanelLayout/PanelLayout';
+import { mocks } from '../Shared/mockData';
+import { NewRequestStepsEnum, PanelTypeEnum } from '../Shared/sharedTypes';
+import { AboutForm } from './Steps/StepOne/AboutForm';
+import { Calculation } from './Steps/StepThree/Calculation';
+import { RentOwn } from './Steps/StepTwo/RentOwn';
+import { StepsList } from './StepsList/StepsList';
+
+export const NewRequestPage: React.FC = () => {
+  const { t } = useTranslation();
+  const steps = useNewStepList();
+
+  const [currentStep, setCurrentStep] = useState(NewRequestStepsEnum.AboutForm);
+  const [percentComplete, setPercentComplete] = useState(25);
```

Comment:
I might start it at 0, as it could seem confusing to the user, they might think they had previously added some information before.